app/templates/content/index.hbs

```diff
@@ -44,6 +44,9 @@
             </div>
         </div> {{!END CONTAINER DIV}}
     </div> {{!END HEADER ROW}}
+    {{#if isAdmin}}
+        {{preprint-status-banner model=model}}
```

Comment:
Within the component, I don't think we should use the name `model`. `preprint=model`, maybe?